_layouts/index.html

```diff
@@ -14,6 +14,7 @@
         <h1 class="fw-bold text-body-emphasis lh-1 mb-3">Kroxylicious, the snappy open source proxy for Apache Kafka<sup>&#174;</sup></h1>
         <p class="lead">Topic encryption, policy-enforcement, multi-tenancy, audit and much more.</p>
         <div class="d-grid gap-2 d-md-flex justify-content-md-start">
+          <a href="{{site.baseurl}}/use-cases/" alt="What can Kroxylicious do" type="button" class="btn btn-light btn-lg px-4 gap-3">What can it do?</a>
```

Comment:
I hear you and I largely agree the thing that tips me over the edge to use cases, is Kroxylicious is just a proxy! If you know what a proxy does in the vaguest terms you have groked the overview hence I want to point people at the next level of information.

I am happy to hear alternative opinions and be out voted. 

Cc @tombentley @katheris @gracegrimwood

Reply:
IMHO I think that 3 buttons looks a bit cluttered (I recognise this is totally a matter of personal preference so I'm happy to be ignored about this), but given that all of these links are already on the page (in the menubar) I don't think it matters too much if we don't highlight all of them in the hero.

In saying that... I think the two most important things to put in the hero are going to be the overview and use cases. When I visit a site for a project I'm unfamiliar with I usually trust that the links in the hero are meant for people who don't know what that project is about and need a "for dummies" version of why they should invest any further time in learning about it, which is really what the overview and use cases are for. The quickstarts and docs are for people who know roughly what Kroxylicious is and have at least a vague idea of why they might want to use it (because if they didn't, they wouldn't bother with any of it), and I think the hero is really meant to catch the attention of people who haven't heard of Kroxylicious before which is a bit of a different audience.

Besides, even if they aren't in the hero, people can still see the links to the quickstarts and docs on every page because they're in the menubar, and if I was someone just wanting to get straight to playing with Kroxylicious I would probably skip the hero content anyway.

Reply:
Ok, call me persuaded. I've updated inline with @gracegrimwood's suggestion